app/components/primer/border_box_component.rb

```diff
@@ -78,7 +88,7 @@ class BorderBoxComponent < Primer::Component
     #         Row one
     #       <% end %>
     #     <% end %>
-    #     <% component.row do %>
+    #     <% component.row(scheme: :yellow) do %>
```

Comment:
I think we could have a new `@example` showcasing all possible colors

Reply:
```suggestion
    #     <% component.row(scheme: :warning) do %>
```